### DISCUSSION TITLE
Install Issues

### DISCUSSION THREAD

**Discussion by cyberawareness** [Security-Onion-Solutions/securityonion#12636] [Unsupported Versions]
I encountered the below problem while installing Security Onion on VMware workingstation Pro 17. The error message says Install had a problem. Please see /root/sosetup.log. A summary of the erros can be found in /root/errors.log. Then I do sudo so-mysql-start which run successfully with now failure, yet I could not ping any IP, seems no internet. I have checked the errors.log, I also attached the content of the errors.log. My system has 32g Ram with 512gb ssd. I selected NAT for fisrt NIC and Host-only for the second NIC. I set IP with CIDR to 192.168.0.X+1/24 being that my router IP is 192.168.0.X (my understanding is that 192.168.0.X+1 to 192.168.0.254 are usbale since my router IP is 192.168.0.X). I check to ensure no device is assigned the IP on the network.  I used 192.168.0.X as gateway IP and I left 8.8.8.8,8.8.4.4 and searchdomain.local as instructed. I chose IP as way to access the web. Kindly help me! I have tried the installation like 20 times the same problem.

![sdf](https://github.com/Security-Onion-Solutions/securityonion/assets/162910389/85541da9-cdda-45ab-909a-608af6c08dd9)

![aaa](https://github.com/Security-Onion-Solutions/securityonion/assets/162910389/757c1305-5401-45f1-b1d8-354e9dc4ee0d)

**Comment by cm-ops** [comment]
If you open your VMware Workstation Virtual Network Editor, what is the subnet address for VMnet type NAT?
